eng/package.ps1

```diff
@@ -57,5 +61,6 @@ main {
             --include=src/** `
             --include=TestRunner.psm1 `
             --include=run-tests.bat `
+            --include=.env `
```

Comment:
I conditional check was added to prevent overwriting local file. Delete would cause same issues with local run. However this script is not meant for local run so I could just remove the file exists condition instead.

Reply:
done